elektronLogging/consoleLogger.go

```diff
@@ -0,0 +1,47 @@
+package elektronLogging
+
+import (
+	log "github.com/sirupsen/logrus"
+	"os"
+	"strings"
+)
+
+type ConsoleLogger struct {
+	LoggerImpl
+}
+
+func NewConsoleLogger(logType int, prefix string) *ConsoleLogger {
+	cLog := &ConsoleLogger{}
+	cLog.Type = logType
+	cLog.SetLogFile(prefix)
+	return cLog
+}
+func (cLog *ConsoleLogger) Log(logType int, level log.Level, logData log.Fields, message string) {
+	if logType <= cLog.Type {
+
+		logger.SetLevel(level)
+
+		logger.SetOutput(os.Stdout)
+		logger.WithFields(logData).Println(message)
+
+		logger.SetOutput(cLog.LogFileName)
+		logger.WithFields(logData).Println(message)
+	}
+	if cLog.next != nil {
+		cLog.next.Log(logType, level, logData, message)
+	}
+}
+
+func (cLog *ConsoleLogger) SetLogFile(prefix string) {
+
+	consoleLogPrefix := strings.Join([]string{prefix, config.ConsoleConfig.FilenameExtension}, "")
+	if logDir != "" {
+		consoleLogPrefix = strings.Join([]string{logDir, consoleLogPrefix}, "/")
+	}
+	if logFile, err := os.Create(consoleLogPrefix); err != nil {
+		log.Fatal("Unable to create logFile: ", err)
+	} else {
+		cLog.LogFileName = logFile
+		cLog.AllowOnConsole = true
```

Comment:
I see why you are directly assigning `true`. However, to allow disabling console logs to stdout retrofit this to `cLog.AllowOnConsole = config.ConsoleConfig.AllowOnConsole`.

Reply:
Sure